testsuite/api/connections.lua

```diff
@@ -90,8 +90,7 @@ printStatus(status, 0)
 -- status:  [correct] error
 -- error:   [addConnection] Type mismatch in connection: sc1.u2 -> sc2.y2
 -- status:  [correct] error
--- error:   [addConnection] Connector is already connected: sc1.u1
--- status:  [correct] error
+-- status:  [wrong] ok
```

Comment:
@arun3688 This looks wrong.

Reply:
@lochel this should be fixed with this d646dbc